button.css

```
@@ -0,0 +1,24 @@
+.button{
+    width: 200px;
+    height: 45px;
+    padding: 13px 36px 15px 36px;
+    text-decoration: none;
+    background: #4985FF;
+    box-shadow: 0px 0px 10px rgba(111, 111, 111, 0.25);
+    border-radius: 30px;
+
+    font-family: Montserrat;
```

здесь к этому шрифту добавь еще sans-serif
```suggestion
    font-family: Montserrat, sans-serif;
```

---

еще рекомендую посмотреть css-переменные — https://developer.mozilla.org/ru/docs/Web/CSS/Using_CSS_custom_properties 

Считается хорошей практикой использовать переменные в местах, где значения повторяются, к примеру это могут быть шрифты, цвета ...